bindata/network/ovn-kubernetes/monitor.yaml

```diff
@@ -23,20 +23,20 @@ spec:
     - openshift-ovn-kubernetes
   selector:
     matchLabels:
-      app: ovnkube-master-metrics
+      app: ovnkube-master
 ---
 apiVersion: v1
 kind: Service
 metadata:
   labels:
-    app: ovnkube-master-metrics
+    app: ovnkube-master
```

Comment:
This should stay as `ovnkube-master-metrics`.

Reply:
After our call yesterday, I tried it but it looks like the ports are not orthogonal. I haven't yet found a clear documentation on Prometheus that explains this. I have asked the monitoring-team to confirm this. So basically the scraping works only if I add a `metrics` port to `ovnkube-db` service as well.

So the way I see it, now we have two options:

1) Either we go back to my previous solution where we sort of umbrella all 6 pods under the `ovn-kubernetes-master-metrics` service and not use the `ovnkube-db` service at all - when it comes to metrics scraping.
2) We use both `ovnkube-db` and `ovn-kubernetes-master-metrics` but then define/expose the 9012 port on both. - I don't know if this will have an (undoing) effect on the TLS work done and all the http/https scraping stuff.